### ISSUE TITLE
PolarisCore

### ISSUE THREAD

**Issue by kazuyabr** [PolarisTeam/PolarisServer#44]
Hi, why I can't see this repository?
I'm Brazilian and I want contribute with this project, but I can't see this project.

**Comment by XenoWarrior** [comment]
> Hi, why I can't see this repository?
> I'm Brazilian and I want contribute with this project, but I can't see this project.

If you look at the read me, it clearly says the repo was moved to another repo. 

Never mind, it was made private or deleted.

**Comment by XenoWarrior** [comment]
Never mind, it was made private

**Comment by HybridEidolon** [comment]
The contents of this repo are still available here; PolarisServer is a different codebase from PolarisCore. PolarisCore was a rewrite of this project. However, PolarisCore has been made private, as it also was largely abandoned.

Given the announcement of PSO2NGS, there may be new developments soon. If it comes to that, I will try to reach out to y'all if you are still interested, @kazuyabr @XenoWarrior.

**Comment by cyberkitsune** [comment]
fwiw @HybridEidolon there's been some additional work to maintain this repo by a few people, so I imagine maybe we should just drop any info about PolarisCore for now?

**Comment by XenoWarrior** [comment]
Thanks for the update, I personally love seeing updates to projects like this, following code that is built from reverse engineering the game!

**Comment by kazuyabr** [comment]
> The contents of this repo are still available here; PolarisServer is a different codebase from PolarisCore. PolarisCore was a rewrite of this project. However, PolarisCore has been made private, as it also was largely abandoned.
> 
> Given the announcement of PSO2NGS, there may be new developments soon. If it comes to that, I will try to reach out to y'all if you are still interested, @kazuyabr @XenoWarrior.

Yes, I have interesting for contribute. You can notify me when to begin the project.

**Comment by kazuyabr** [comment]
> fwiw @HybridEidolon there's been some additional work to maintain this repo by a few people, so I imagine maybe we should just drop any info about PolarisCore for now?

I believe so, because the more obsolete the original project is (in the case of PSO2 Sega.jp) the more people will be free to produce content and thus create projects and private servers (or pirates based on that project) without problems with legal rights to use.
I think of this following a logic used for Lineage 2 servers, both done in JAVA and done in C. Nobody is concerned with copyright in this project since its versions are obsolete in relation to the original from NCSoft.
